__tests__/support/gql/index.js

```diff
@@ -88,6 +89,10 @@ const root = {
       id,
     };
   },
+  updateUser({ id, input }) {
+    const user = new User(id, userDatabase[1]);
```

Comment:
I think we should change the `userDatabase` to only contain `User` instances. Something like 
```
const userDatabase = {
  1: new User(1, {
    id: 1,
    firstName: 'John',
    lastName: 'Doe',
    age: 30,
    username: 'test@test.test',
    password: 'test1234',
  }),
};
```

Then we don't need to use `new User()` to get the instance.
Think this `userDatabase[1]` should be `userDatabase[id]`,